jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt

```diff
@@ -18,7 +19,7 @@ import org.jitsi.meet.sdk.JitsiMeetConferenceOptions
 /**
  * Activity extending JitsiMeetActivity in order to override the conference events
  */
-class JitsiMeetPluginActivity : JitsiMeetActivity() {
+class JitsiMeetPluginActivity<Activity> : JitsiMeetActivity() {
```

Comment:
I also did not add this and the app built successfully. Can you check if this is needed?

Reply:
that's very strange if that works for you. i was getting this error on compile time and after some research i found the cause

Reply:
btw this is not a crucial change in itself as well. it's just enforcing the type for the class. so it creates no harm in there.